pandas/io/excel/_openpyxl.py

```diff
@@ -61,17 +61,23 @@ def __init__(
             self.sheets = {name: self.book[name] for name in self.book.sheetnames}
 
         else:
-            # Create workbook object with default optimized_write=True.
-            self.book = Workbook()
+            try:
+                # Sheets are not automatically created in the workbook
+                self.book = Workbook(write_only=True)
+            except ImportError:
+                print("Warning: lxml is not installed")
+                print("Memory usage may be much higher.")
+                self.book = Workbook()
 
-            if self.book.worksheets:
-                self.book.remove(self.book.worksheets[0])
+                if self.book.worksheets:
+                    self.book.remove(self.book.worksheets[0])
 
     def save(self):
         """
         Save workbook to disk.
         """
-        self.book.save(self.handles.handle)
+        # TODO: Handle errors from saving more than once
```

Comment:
What are we supposed to do with this TODO? Are you fixing it later in this PR, or do we need to create an issue?

Reply:
That was a TODO for me! I'm committing it today.

Reply:
> What are we supposed to do with this TODO? Are you fixing it later in this PR, or do we need to create an issue?

That was a TODO for me! I'm committing it today.